fed/mosaic-sumo/src/main/java/org/eclipse/mosaic/fed/sumo/util/SumoRouteFileCreator.java

```diff
@@ -59,7 +59,7 @@
 
     private final static Logger log = LoggerFactory.getLogger(SumoRouteFileCreator.class);
 
-    final static String VEHICLE_TYPE_ROUTE_FILE_NAME = "mosaic_types.rou.xml";
+    final static String MOSAIC_TYPES_FILE_NAME = "mosaic_types.add.xml";
```

Comment:
Now you could rename the class 😄 , e.g. `SumoVehicleTypesWriter`

Reply:
I like.